<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageFilterWindow</class>
 <widget class="QMainWindow" name="ImageFilterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>按尺寸筛选图像</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>图片路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtPath"/>
      </item>
      <item>
       <widget class="QToolButton" name="btnOpen">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>图像尺寸(宽*高)：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbbWidth">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>*</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbbHeight">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>匹配类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdSize">
        <property name="text">
         <string>匹配宽高</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdSizeLage">
        <property name="text">
         <string>宽高均大于等于当前值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdWidthOrHeight">
        <property name="text">
         <string>匹配宽或高</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdWidth">
        <property name="text">
         <string>只匹配宽度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdHeight">
        <property name="text">
         <string>只匹配高度</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>图像方向：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdIgnore">
        <property name="text">
         <string>忽略</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdHor">
        <property name="text">
         <string>横向</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdVer">
        <property name="text">
         <string>纵向</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>503</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCheck">
         <property name="text">
          <string>筛选</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCopyTo">
         <property name="text">
          <string>复制到</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMoveTo">
         <property name="text">
          <string>移动到</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="lvData">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
